app/javascript/components/molecules/Dropdown.js

```diff
@@ -102,7 +102,7 @@ class Dropdown extends React.Component {
       <button
         aria-expanded={this.state.expanded ? 'true' : 'false' }
         onClick={this.toggleExpand} ref={(node) => this.buttonRef = node }
-        className={`button dropdown-button has-icon--right text-align-left ${buttonClasses.join(' ')}`}
+        className={`button dropdown-button has-icon--right text-align-left ${buttonClasses ? buttonClasses.join(' ') : ''}`}
```

Comment:
This could be slightly improved using https://github.com/JedWatson/classnames like:
`className={classNames( 'button dropdown-button has-icon--right text-align-left', buttonClasses)}`

Reply:
thanks for the suggestion! i think in this case because the functionality is pretty trivial, i’d like to err on the side of using fewer dependencies. but good to know that package is out there in case we need more complex functionality in this area!